pkg/plugins/platform/utils.go

```diff
@@ -23,7 +23,7 @@ func GetApps() map[string][]App {
 	apps := make(map[string][]App)
 	for platformType := range PluginMap {
 		labels := map[string]string{
-			"com.ddev.platform":          platformType,
+			"com.ddev.platform":          "ddev",
```

Comment:
The com.ddev.platform label now contains "ddev", so we're searching for it that way. It used to be "local", but the idea here is to change all those.